noxfile.py

```diff
@@ -514,3 +520,78 @@ def prerelease_deps(session):
             system_test_folder_path,
             *session.posargs,
         )
+
+
+def install_conda_unittest_dependencies(session, standard_deps, conda_forge_packages):
+    """Installs packages from conda forge, pypi, and locally."""
+
+    # Install from conda-forge and default conda package repos.
+    session.conda_install(*conda_forge_packages, channel=["defaults", "conda-forge"])
```

Comment:
If both `default` and `conda-forge` have a package, would we get the version from `conda-forge` or `default`? IMO, we should prefer `conda-forge` since those packages are updated more frequently in general.

Reply:
Good question. I don't know the answer yet. I will look it up and see if there is a way to identify a preference for source.

Not gonna hold up the PR for this though. If a change is needed, will add to a separate PR.